cmd/utils/flags.go

```diff
@@ -491,6 +493,12 @@ var (
 		Usage:    "Enable recording the SHA3/keccak preimages of trie keys",
 		Category: flags.PerfCategory,
 	}
+	CacheLogCacheSizeFlag = &cli.IntFlag{
+		Name:     "cache.logcache",
+		Usage:    "Size of the block logs cache (for RPC filters)",
```

Comment:
What does "size" mean? Number of blocks, number of txs, number of logs, bytes, percentage of the total cache allowance?

Reply:
It means number of blocks right now. Need to expand this description.